README.rst

```diff
@@ -105,11 +103,11 @@ Forest software development kit (SDK):
 
 * `Forest SDK <https://pyquil-docs.rigetti.com/en/stable/>`__
 
-Alternatively, you may sign up for Rigetti's Quantum Cloud Services (QCS) to acquire a Quantum Machine
-Image (QMI) which will allow you to compile your quantum code and run on real quantum processing units (QPUs),
-or on a preinstalled QVM. Note that this requires a valid QCS account.
+Alternatively, you may sign up for Rigetti's Quantum Cloud Services (QCS)  which will allow you to compile your 
```

Comment:
You've already defined the abbreviation, so just use it (`QCS`) instead of the expansion. (Likewise below for `QPU`.)